delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/resources/Resources.scala

```diff
@@ -355,22 +353,29 @@ object Resources {
 
     def stateWhereTagExistsOnResource(c: ModifyCommand, tag: UserTag) = {
       stateWhereResourceExists(c).flatMap { s =>
-        IO.raiseWhen(!s.tags.contains(tag))(TagNotFound(tag)).as(s)
+        raiseWhenDifferentSchema(c, s) >>
```

Comment:
Not in the scope of this PR but we could consider dropping the check on the schema for commands where the schema is not relevant (ex: tag/refresh) and even the fetch operation